bin/cli.js

```diff
@@ -42,6 +42,10 @@ var ghauthOpts = {
   authUrl: isEnterprise ? argv.endpoint.replace(/\/+$/, '') + '/authorizations' : null
 }
 
+if (argv.skipcertcheck) {
+  process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0
```

Comment:
writing to `process.env` is usually a sign your program is doing something wrong -- i think this might be an issue outside the scope of a fix like this, or possibly outside the scope of `gh-release`. will have to research your issue more first

Reply:
Yeah def hacky but couldn't find another way to work around it and I assume others might need an escape hatch also and it's optional.  No worries feel free to close if it's not in line with the project or whatever.